src/module_wrap.cc

```diff
@@ -142,6 +142,57 @@ v8::Maybe<bool> ModuleWrap::CheckUnsettledTopLevelAwait() {
   return v8::Just(false);
 }
 
+static Local<Object> createImportAttributesContainer(
+    Realm* realm,
+    Isolate* isolate,
+    Local<FixedArray> raw_attributes,
+    const int elements_per_attribute) {
+  CHECK_EQ(raw_attributes->Length() % elements_per_attribute, 0);
+  Local<Object> attributes =
+      Object::New(isolate, v8::Null(isolate), nullptr, nullptr, 0);
```

Comment:
You can collect the keys and values into two std::vectors and then pass them into the `Object::New` at one go, that can be faster than doing `obj->Set` repeatedly.